app/presenters/coronavirus/sub_section_json_presenter.rb

```diff
@@ -50,14 +44,6 @@ def sub_sections
     content_groups.map { |content_group| sub_section_hash_from_content_group(content_group) }
   end
 
-  def link_set_as_featured?
```

Comment:
I think you still need to do this so that the link has the `featured` flag set to true in the publishing-api payload, e.g.

```
{
  url: "/getting-tested-for-coronavirus?priority-taxon=774cee22-d896-44c1-a611-e3109cce8eae",
  label: "Find out about coronavirus testing",
  description: "Find out whether you need a PCR test or rapid lateral flow test for coronavirus (COVID-19)",
  featured_link: true
}
```

Reply:
as far as we can tell the only place that that method was being used was here https://github.com/alphagov/collections-publisher/pull/1299/files#diff-221c8bf8dbf2b355bc4c0e9cb9ee9e8424bd94bbd3078a19efea804a65d9bba6L33 in order to check whether we needed to add extra errors

Reply:
As far as we could tell the only place that this was in the validation in the json presenter which has now been removed. https://github.com/alphagov/collections-publisher/pull/1299/files#diff-221c8bf8dbf2b355bc4c0e9cb9ee9e8424bd94bbd3078a19efea804a65d9bba6L33

It's not being used anywhere else in the repo so i think it's safe